components/build-templates/base/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 resources:
 - 0-namespace.yaml
 - e2e/
+- rbac
```

Comment:
I think you could put the build-admin.yaml file in the e2e/ folder, since it's related only to the build-templates-e2e namespace

Reply:
moved it to the e2e folder and changed the kustomization files

Reply:
Looking at the new diff, maybe just edit the existing role.yaml and rolebinding.yaml? :sweat_smile: 

```
- rolebinding.yaml
- role.yaml
- rbac
```

Reply:
yeah..that makes sense :D